commit 886e00c5fe3d41ef161f5fa04b5dc1cc89a1bead
Author: Adam Ford <aford173@gmail.com>
Date:   Sun Oct 28 15:13:48 2018 -0500

    ARM: dts: am3517: Fix pinmuxing for CD on MMC1
    
    [ Upstream commit e7f4ffffa972db4820c23ff9831a6a4f3f6d8cc5 ]
    
    The MMC1 is active low, not active high.  For some reason,
    this worked with different combination of U-Boot and kernels,
    but it's supposed to be active low and is currently broken.
    
    Fixes: cfaa856a2510 ("ARM: dts: am3517: Add pinmuxing, CD and
    WP for MMC1") #kernel 4.18+
    
    Signed-off-by: Adam Ford <aford173@gmail.com>
    Signed-off-by: Tony Lindgren <tony@atomide.com>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/arm/boot/dts/am3517-evm.dts b/arch/arm/boot/dts/am3517-evm.dts
index 1d158cfda15f..c45aef806870 100644
--- a/arch/arm/boot/dts/am3517-evm.dts
+++ b/arch/arm/boot/dts/am3517-evm.dts
@@ -227,7 +227,7 @@
 	vmmc-supply = <&vmmc_fixed>;
 	bus-width = <4>;
 	wp-gpios = <&gpio4 30 GPIO_ACTIVE_HIGH>; /* gpio_126 */
-	cd-gpios = <&gpio4 31 GPIO_ACTIVE_HIGH>; /* gpio_127 */
+	cd-gpios = <&gpio4 31 GPIO_ACTIVE_LOW>; /* gpio_127 */
 };
 
 &mmc3 {
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1026_886e00c5fe3d41ef161f5fa04b5dc1cc89a1bead.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1026_886e00c5fe3d41ef161f5fa04b5dc1cc89a1bead.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
